templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/AddCommunityForm.vue

```diff
@@ -77,9 +94,16 @@ export default {
       showPicker: false,
       isLoading: false,
       isInputTouched: false,
+      emojiImg: null,
     }
   },
   computed: {
+    emojiDataAll() {
+      return EmojiAllData
```

Comment:
It's nicer if these names are kept consistent
```suggestion
    emojiAllData() {
      return EmojiAllData
```

Reply:
Renamed